criu/cr-restore.c

```diff
@@ -2360,6 +2362,25 @@ static int restore_root_task(struct pstree_item *init)
 	if (ret < 0)
 		goto out_kill;
 
+	for_each_pstree_item(item) {
+		if (task_alive(item) && (item->nr_threads > 1)) {
```

Comment:
That's making the code harder to read. Make a reversed condition and call `continue` inside.